packages/bruno-app/src/components/RequestTabs/RequestTab/ConfirmCollectionClose/index.js

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
```

Comment:
I see there is a lot of repetition across `ConfirmCollectionClose` , ConfirmFolderClose`, and `ConfirRequestClose`
This is fine. Lets not try to modularise this. 

A good read - https://overreacted.io/the-wet-codebase/